Documentation/README_ConstraintManager.md

```diff
@@ -0,0 +1,162 @@
+
+<img src="../Documentation/Images/ObjectManipulator/MRTK_Constraint_FixedRotationToUser.gif" width="400">
+
+This constraint fixes the relative rotation between the user and the manipulated object while it is being manipulated. This is useful for slates or panels as it ensures that the manipulated object always shows the same face to the user as it did at the start of manipulation. The `FixedRotationToUserConstraint` does not have any unique properties.
+
+### FixedRotationToWorldConstraint
+
+<img src="../Documentation/Images/ObjectManipulator/MRTK_Constraint_FixedRotationToWorld.gif" width="400">
+
+This constraint fixes the global rotation of the manipulated object while it is being manipulated. This can be useful in cases where no rotation should be imparted by manipulation. The `FixedRotationToWorldConstraint` does not have any unique properties:
+
+### MaintainApparentSizeConstraint
+
+<img src="../Documentation/Images/ObjectManipulator/MRTK_Constraint_MaintainApparentSize.gif" width="400">
+
+When this constraint is attached to an object, no matter how far the object is from the user, it will maintain the same apparent size to the user (i.e. it will take up the same proportion of the user's field of view). This can be used to ensure that a slate or text panel remains readable while manipulating. The `MaintainApparentSizeConstraint` does not have any unique properties:
+
+### MoveAxisConstraint
+
+<img src="../Documentation/Images/ObjectManipulator/MRTK_Constraint_MoveAxis.gif" width="400">
+
+This constraint can be used to fix along which axes a manipulated object can be moved. This can be useful for manipulating objects over the surface of a plane, or along a line. The properties for `MoveAxisConstraint` are as follows:
+
+#### Constraint on movement
+
+Specifies which axes to prevent movement on. By default these axes will be global rather than local, but this can be changed below. Because this property is a flag, any number of options can be selected.
+
+* *X Axis*: Movement along the x-axis is constrained if selected.
+* *Y Axis*: Movement along the y-axis is constrained if selected.
+* *Z Axis*: Movement along the z-axis is constrained if selected.
+
+#### Use local space for constraint
+
+Will constrain relative the manipulated object's local transform axes if true. False by default.
+
+### RotationAxisConstraint
+
+<img src="../Documentation/Images/ObjectManipulator/MRTK_Constraint_RotationAxis.gif" width="400">
+
+This constraint can be used to fix about which axes a manipulated object can be rotated. This can be useful for keeping a manipulated object upright, but still allowing y-axis rotations, for example. The properties for `RotationAxisConstraint` are as follows:
+
+#### Constraint on rotation
+
+Specifies which axes to prevent rotation about. By default these axes will be global rather than local, but this can be changed below. Because this property is a flag, any number of options can be selected.
```

Comment:
```suggestion
Specifies which axes to prevent rotation about. By default. these axes will be global rather than local, but this can be changed below. Because this property is a flag, any number of options can be selected.
```